atd-vzd/migrations/default/1725990787849_drvr_lic_type_id/up.sql

```diff
@@ -0,0 +1,123 @@
+create table lookups.drvr_lic_type (
+    id integer primary key,
+    label text not null unique,
```

Comment:
i always forget that you can name constraints implicitly like this. Nice that it takes away the headache of naming yet another thing!